tests/common/runner.rs

```diff
@@ -29,14 +30,21 @@ pub fn assert_execution_result(sequential_result: &PevmResult, parallel_result:
 // the execution results match.
 pub fn test_execute_revm<S: Storage + Clone + Send + Sync>(
     storage: S,
+    chain: Chain,
```

Comment:
The call sites are getting quite busy so let's do the other way around instead -- remove `chain`, `spec_id`, and `block_env` from call sites and hardcode the default values here.

We already have better tests for different specs in the Ethereum tests and real blocks via the Alloy interface.